README.md

```diff
@@ -25,39 +25,6 @@ npm run rewrite PATH
 
 where `PATH` is a file or directory of files to be rewritten.
```

Comment:
Have you tested that this works, or do you need `npm run rewrite -- PATH`? I'm never sure when you need the `--` with `npm run` or not.

Reply:
Actually I wonder if it'd be simpler to just tell people to do `node bin/index.js`, and then omit the `rewrite` script in package.json entirely. Now that we don't have the jscodeshift indirection, using the npm script stuff becomes much less necessary.

Reply:
It works as `npm run rewrite PATH` at the moment. We could tell people to do either, I don't know what the convention with using `npm run` is

Reply:
I think removing the rewrite script makes sense at this point.

Reply:
Ok I can take it out, should I change the name to `rewrite.js` or something more semantic?

Reply:
Yeah that's probably a good idea.

Reply:
You only need `--` if your script is taking flags. Without the `--` the flags would be parsed by npm's flag parser. So since rewrite.js doesn't do any flag parsing (yet) the `--` are not needed.